_docs-sources/pipelines/upgrading/index.md

```diff
@@ -0,0 +1,166 @@
+
+Once you have reviewed the `plan` output and gotten any necessary approvals, merge the PR. The EDR version of pipelines will run `apply` to create the resources.
+
+## Setting up secrets
+
+Modern Gruntwork Pipelines requires the use of GitHub personal access tokens (PAT) to allow workflows in the `infrastructure-live` to run workflows in the `infrastructure-pipelines` repository and workflows in the `infrastructure-pipelines` repository access to the code in `infrastructure-live` as well as repos for the Gruntwork Library. This section will guide you in establishing secrets in both repositories and comprehending the security implications of the multi-PAT configuration.
+
+### Infrastructure live
+
+Navigate to your `infrastructure-live` repository (it may be named `<your-company>-infrastructure-live`). Select the **Settings** tab, select the **Secrets and variables** drop down on the left side panel, then select **Actions**. Create three secrets named `GRUNTWORK_CODE_ACCESS_TOKEN`, `PIPELINES_DISPATCH_TOKEN`, and `INFRA_LIVE_ACCESS_TOKEN`. Use the corresponding value from the tokens you created in [creating tokens for workflows](#creating-tokens-for-workflows) for the secret value.
+
+### Infrastructure pipelines
+
+Navigate to the `infrastructure-pipelines` repository. Select the **Settings** tab, select the **Secrets and variables** drop down on the left side panel, then select **Actions**. Create two secrets named `INFRA_LIVE_ACCESS_TOKEN` and `GRUNTWORK_CODE_ACCESS_TOKEN`. Use the corresponding value from the tokens you created in [creating tokens for workflows](#creating-tokens-for-workflows) for the secret value.
+
+## Updating your accounts file
+
+As a part of separate product improvement the `accounts.json` file has been changed to `accounts.yml`. In addition to readability improvements, we found that Terraform's built-in YAML formatters better supported our use case of re-generating this file to support adding new accounts using GitOps workflows. There is no functional difference between the two different file types in the EDR version of pipelines or Modern Gruntwork Pipelines.
+
+As part of a separate product improvement initiative, we have transitioned from the `accounts.json` file to `accounts.yml`. This change not only enhances readability but is also functionally better, with Terraform's built-in YAML formatter better meeting our requirement for regenerating a formatted file to facilitate the addition of new accounts through GitOps workflows. It's important to note that there is no functional distinction between these two file types in either the EDR version of pipelines or Modern Gruntwork Pipelines.
```

Comment:
this line is mostly duplicated from above

Reply:
Fixed in [73295b2](https://github.com/gruntwork-io/docs/pull/1115/commits/73295b26e2fbe7ab38f0addf4381fb0d3a67be12)